README.md

```diff
@@ -198,7 +201,13 @@ facet configuration table is available here.
 | clearFacetsSelector | String | Class name for button to clear the selected filters under a facet |
 | removeFacetsSelector | String | for deleting each selected filter under a facet. this class name is must to work |
 | onFacetLoad | function | if you wish to do something after each facet selection |
-| applyMultipleFilters | Boolean | if you wish to apply multiple filters together
+| applyMultipleFilters | Boolean | if you wish to apply multiple filters together |
+| isCollapsible | Boolean | if you wish to have accordian on facets |
+| defaultOpen | String | Available options ALL , FIRST , NONE |
+| isSearchable | Boolean | if you wish to have a search feature on facets |
+| searchPlaceHolder | String | placeholder for the facet search input |
+| textFacetWrapper | String | css class for the facets list |
```

Comment:
if it is CSS class please reflect that in the config name. Also, why is it called `textFacetWrapper` if it is CSS class for facets list? Shouldn't it be called `facetsWrapperClass` instead ?